app/controllers/api/databases.php

```diff
@@ -2227,6 +2227,30 @@ function updateAttribute(
         $dbForProject->deleteCachedDocument('database_' . $db->getInternalId(), $collectionId);
         $dbForProject->deleteCachedCollection('database_' . $db->getInternalId() . '_collection_' . $collection->getInternalId());
 
+        if ($attribute->getAttribute('type') === Database::VAR_RELATIONSHIP) {
+            $options = $attribute->getAttribute('options');
+            if ($options['twoWay']) {
+                $relatedCollection = $dbForProject->getDocument('database_' . $db->getInternalId(), $options['relatedCollection']);
+
+                if ($relatedCollection->isEmpty()) {
+                    throw new Exception(Exception::COLLECTION_NOT_FOUND);
+                }
+
+                $relatedAttribute = $dbForProject->getDocument('attributes', $db->getInternalId() . '_' . $relatedCollection->getInternalId() . '_' . $options['twoWayKey']);
+
+                if ($relatedAttribute->isEmpty()) {
+                    throw new Exception(Exception::ATTRIBUTE_NOT_FOUND);
+                }
+
+                if ($relatedAttribute->getAttribute('status') === 'available') {
```

Comment:
@abnegate should we check 'available' for the related on twoKey = true?

Reply:
Yep I think so :+1: